android-kit-base/src/main/java/com/mparticle/kits/KitManagerImpl.java

```diff
@@ -1334,12 +1334,25 @@ public void execute(String viewName,
             try {
                 if (provider instanceof KitIntegration.RoktListener && !provider.isDisabled()) {
                     MParticleUser user = MParticle.getInstance().Identity().getCurrentUser();
-                    Map<String, Object> objectAttributes = new HashMap<>();
 
+                    JSONArray jsonArray = provider.getConfiguration().getPlacementAttributes();
+
+                    for (int i = 0; i < jsonArray.length(); i++) {
+                        JSONObject obj = jsonArray.optJSONObject(i);
+                        if (obj == null) continue;
+
+                        String mapKey = obj.optString("map");
+                        String valueValue = obj.optString("value");
+                        if (attributes.containsKey(mapKey)) {
+                            String value = attributes.remove(mapKey);
+                            attributes.put(valueValue, value);
```

Comment:
maybe this might be more readable?
`mapkey` to `mapFrom`
`valueValue` to `mapTo`